odoo/addons/base/models/ir_actions.py

```diff
@@ -65,6 +68,22 @@ def unlink(self):
         self.clear_caches()
         return res
 
+    def _ensure_group_xml_id(self):
+        """Create an external identifier to the groups used in the actions
+        This is required for the cache of get_bindings"""
+        if 'groups_id' in self._fields:
+            self.env['ir.model.data'].create(
+                [
+                    {
+                        'name': f'group_{group_id}',
+                        'model': 'res.groups',
+                        'res_id': group_id,
+                        'module': '__export__',
```

Comment:
Maybe use a signalling / fake module other than export since this has nothing to do with export?

Reply:
Indeed but I have no idea which fake name I could use, besides __export__.

Reply:
I think anything which is not a module name would work.

Using a real module name would cause the records to be deleted if the corresponding module is uninstalled. Which is likely why the export uses a dunder name. The dunder has no specific meaning, but it seems unlikely a module would actually be called `__export__` (and if it is only god can help you).